src/Microsoft.Azure.WebJobs.Logging/IFunctionDefinition.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.WindowsAzure.Storage.Table;
+using System;
+using System.Globalization;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    /// <summary>
+    /// Information about a function definition. 
+    /// </summary>
+    public interface IFunctionDefinition
```

Comment:
We'll probably want to review all the public APIs for logging. Interfaces can't be extended in the future, and we know we're going to want to add more things here.